.github/workflows/test.yml

```diff
@@ -26,7 +26,7 @@ jobs:
       image: openknowledge/ckan-dev:${{ matrix.ckan-version }}
     services:
       solr:
-        image: ckan/ckan-solr-dev:${{ matrix.ckan-version }}
+        image: ckan/ckan-solr:${{ matrix.ckan-version }}
```

Comment:
```suggestion
        image: ckan/ckan-solr:${{ matrix.ckan-version }}-solr9
```

Reply:
We are using this variant to future proof the image name. Other than that looks good, feel free to merge